src/main/java/tech/blacksource/blacknectar/service/Server.java

```diff
@@ -51,22 +54,25 @@
     private final Aroma aroma;
     private final SayHelloOperation sayHelloOperation;
     private final GetSampleStoreOperation getSampleStoreOperation;
+    private final GetStatesOperation getStatesOperation;
```

Comment:
This list of operations is getting crazy and will continue to get crazier.

I suggest creating a layer of abstraction over setting up each individual routes. This way the server gets to focus on the higher-level elements.